sessions/react-effects-and-fetch/pokemon-api/src/components/PokemonList.css

```diff
@@ -0,0 +1,12 @@
+.button {
+    border: none;
+    padding-inline: 20px;
+    padding-block: 10px;
+    background-color: var(--primary);
```

Comment:
`primary-light`

Reply:
why? this is not the journal challenge...